samples/sample_multi_transcode/src/transcode_utils.cpp

```diff
@@ -139,6 +139,12 @@ void TranscodingSample::PrintHelp()
     msdk_printf(MSDK_STRING("                      -hw - platform-specific on default display adapter (default)\n"));
     msdk_printf(MSDK_STRING("                      -hw_d3d11 - platform-specific via d3d11\n"));
     msdk_printf(MSDK_STRING("                      -sw - software\n"));
+#if defined(LINUX32) || defined(LINUX64)
+    msdk_printf(MSDK_STRING("   -device /path/to/device - set graphics device for processing\n"));
+    msdk_printf(MSDK_STRING("                              For example: '-device /dev/dri/card0'\n"));
+    msdk_printf(MSDK_STRING("                                           '-device /dev/dri/renderD128'\n"));
+    msdk_printf(MSDK_STRING("                              if not specified, the first Intel device sets by default\n"));
```

Comment:
better: "If not specified, defaults to the first Intel device found on the system"

Reply:
Done